src/wp-admin/includes/class-walker-nav-menu-edit.php

```diff
@@ -262,6 +262,29 @@ public function start_el( &$output, $data_object, $depth = 0, $args = null, $cur
 					<button type="button" class="button-link menus-move menus-move-top" data-dir="top"><?php _e( 'To the top' ); ?></button>
```

Comment:
Adding an additional button here that is used to toggle the visibility of position settings would be good. E.g. 'Set Position'.

Reply:
@joedolson Instead of toggle visibility of position settings here, can't we just add it in screen options. That make sense. Let me know what you think.

Reply:
Seeing it now, I feel like maybe we don't really need to hide it at all. It seems totally reasonable to me as is. There could be a future option to add it in screen options, to optionally hide it; but to me this seems easily useful enough that I wouldn't prioritize it.

Reply:
I am thinking of changing the order, and moving them above the button movers. That'll group all of the form fields above all of the non-form field options.